libraries/GCS_MAVLink/GCS.h

```diff
@@ -512,6 +512,23 @@ class GCS_MAVLINK
     virtual bool persist_streamrates() const { return false; }
     void handle_request_data_stream(const mavlink_message_t &msg);
 
+    AP_Int16 options;
+    enum class Option : uint16_t {
+        MAVLINK2_SIGNING_DISABLED = (1U << 0),
+        NO_FORWARD                = (1U << 1),  // don't forward MAVLink data to or from this device
+        NOSTREAMOVERRIDE          = (1U << 2),  // don't allow GCS to override streamrates
```

Comment:
It's not just GCS is it? I think this would also disable stream rate requests from another vehicle, a thing I have done.

Reply:
Reworded, thanks!